packages/graphql-auth-transformer/src/__tests__/__snapshots__/OperationsArgument.test.ts.snap

```diff
@@ -5,15 +5,7 @@ exports[`Test "create", "update", "delete" auth operations 1`] = `"$util.toJson(
 exports[`Test "create", "update", "delete" auth operations 2`] = `"$util.toJson($ctx.result)"`;
 
 exports[`Test "create", "update", "delete" auth operations 3`] = `
-"## [Start] Set default values. **
-$util.qr($context.args.input.put(\\"id\\", $util.defaultIfNull($ctx.args.input.id, $util.autoId())))
-#set( $createdAt = $util.time.nowISO8601() )
-## Automatically set the createdAt timestamp. **
-$util.qr($context.args.input.put(\\"createdAt\\", $util.defaultIfNull($ctx.args.input.createdAt, $createdAt)))
-## Automatically set the updatedAt timestamp. **
-$util.qr($context.args.input.put(\\"updatedAt\\", $util.defaultIfNull($ctx.args.input.updatedAt, $createdAt)))
-## [End] Set default values. **
-## [Start] Determine request authentication mode **
+"## [Start] Determine request authentication mode **
```

Comment:
Please rebase this should not have changed here

Reply:
done